lib/code-manager.js

```diff
@@ -38,56 +46,62 @@ export function getRows(startRow, endRow) {
   return normalizeString(code);
 }
 
-export function getSelectedText() {
-  return normalizeString(store.editor.getSelectedText());
+export function getSelectedText(editor: ?atom$TextEditor) {
+  if (editor) return normalizeString(editor.getSelectedText());
 }
 
-export function isComment(position) {
-  const scope = store.editor.scopeDescriptorForBufferPosition(position);
+export function isComment(editor: atom$TextEditor, position: atom$Point) {
+  const scope = editor.scopeDescriptorForBufferPosition(position);
   const scopeString = scope.getScopeChain();
   return _.includes(scopeString, "comment.line");
 }
 
-export function isBlank(row) {
-  return store.editor.getBuffer().isRowBlank(row) ||
-    store.editor.languageMode.isLineCommentedAtBufferRow(row);
+export function isBlank(editor: atom$TextEditor, row: number) {
+  return editor.getBuffer().isRowBlank(row) || editor.isBufferRowCommented(row);
```

Comment:
👍  for using the documented Atom API here.

I'm confident that it does the right thing. I assume you tested it 😄